src/uwtools/set_config.py

```diff
@@ -120,6 +119,24 @@ def create_config_obj(argv):
 
         config_obj.update_values(user_config_obj)
 
+    if user_args.values_needed:
+        config_dict = config_obj.data
+        set_var = []
+        jinja2_var = []
+        undeclared_var = []
+        config.Config.iterate_values(config_obj, config_dict, set_var, jinja2_var, undeclared_var)
+
+        log.info('Filled template variables:')
+        for var in set_var:
+            log.info(var)
+        log.info('Variables left as jinja2 templates:')
+        for var in jinja2_var:
+            log.info(var)
+        log.info('Values still needed:')
+        for var in undeclared_var:
+            log.info(var)
+        return
```

Comment:
For these section headers, could we change them to be a little clearer and more readable? Perhaps...

- Keys that are complete
- Keys that have unfilled jinja2 templates
- Keys that are set to empty

Also, if there's a way to highlight the key lists in the output it would be good. Either indent the list, or have a horizontal line after the header, maybe?